packages/mobile/src/escrow/EscrowedPaymentListScreen.tsx

```diff
@@ -43,57 +38,31 @@ const mapStateToProps = (state: RootState): StateProps => ({
   recipientCache: recipientCacheSelector(state),
 })
 
-type Props = WithNamespaces & StateProps & DispatchProps
-
-export class EscrowedPaymentListScreen extends React.Component<Props> {
-  static navigationOptions = () => ({
-    ...headerWithBackButton,
-    headerTitle: i18n.t('inviteFlow11:pedningInvitations'),
-  })
+type Props = NavigationInjectedProps & WithNamespaces & StateProps & DispatchProps
 
-  renderRequest = (payment: EscrowedPayment, key: number, allPayments: EscrowedPayment[]) => {
-    return (
-      <View key={key}>
-        <EscrowedPaymentListItem payment={payment} />
-        {key < allPayments.length - 1 && <View style={styles.separator} />}
-      </View>
-    )
-  }
+export const listItemRenderer = (payment: EscrowedPayment, key: number | undefined = undefined) => {
+  return (
+    <View key={key}>
+      <EscrowedPaymentListItem payment={payment} />
+    </View>
+  )
+}
 
-  render() {
-    return (
-      <SafeAreaView style={styles.container}>
-        <DisconnectBanner />
-        <PaymentRequestBalance dollarBalance={this.props.dollarBalance} />
-        <SectionHeader text={this.props.t('payments')} />
-        {this.props.sentEscrowedPayments.length > 0 ? (
-          <ScrollView>
-            <View style={[componentStyles.roundedBorder, styles.scrollArea]}>
-              {this.props.sentEscrowedPayments.map(this.renderRequest)}
-            </View>
-          </ScrollView>
-        ) : (
-          <PaymentRequestListEmpty />
-        )}
-      </SafeAreaView>
-    )
-  }
+const EscrowedPaymentListScreen = (props: Props) => {
+  const { dollarBalance, navigation } = props
+  useBalanceInNavigationParam(dollarBalance, navigation)
+  return (
+    <NotificationList<EscrowedPayment>
```

Comment:
Can't TypeScript infer that type that from items?

Reply:
It does indeed! I've underestimated TS :)